templates/taxi/car_detail.html

```diff
@@ -1,12 +1,15 @@
 {% extends "base.html" %}
 
 {% block content %}
+  <h1>{{ car.model }}</h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
+  <h3>Drivers</h3>
   <hr>
   <ul>
   {% for driver in car.drivers.all %}
-    <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
+    <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>    
   {% endfor %}
-  </ul>
+  </ul> 
+  <a class="btn btn-primary" href="{% url 'taxi:car-update' pk=car.id %}">Update this car information</a><br><br>
+  <a class="btn btn-primary" href="{% url 'taxi:car-delete' pk=car.id %}">Delete this car</a>
```

Comment:
According to the checklist, you should add a 'Cancel' button apart from the 'Delete' one. The 'Cancel' button should lead to the previous page the user was on.